doc/contributing.rst

```diff
@@ -175,6 +175,7 @@ MNE-Python specific coding guidelines
 * Avoid unnecessary cosmetic changes if they are not the goal of the PR, this will help keep the diff clean and facilitate reviewing.
 * Use underscores to separate words in non class names: n_samples rather than nsamples.
 * Use CamelCase for class names.
+* Import packages in the order: builtin modules, standard scientific modules, other modules, mne
```

Comment:
just mention in parens that standard scientific modules are `numpy as np` and then `scipy` submodules or individual imports`. This probably would look better as an indented list, too.